### PR TITLE
Stub out html for get_involved page

### PR DESCRIPTION
**Pull request by suttondemlong**
I deleted an unused impact page file and stubbed out the HTML for the Get Involved Page. Let me know if I missed anything or if the HTML semantics can be improved.

### CODE REVIEW

File: gworv/templates/involvement/get_involved_index_page.html [hunk 1]
```diff
@@ -1,5 +1,61 @@
 {% extends "base.html" %} 
 
+{% load wagtailcore_tags %}
+{% load wagtailimages_tags %}
+
+{% block body_class %}template-getinvolvedindexpage{% endblock %}
+
 {% block content %}
-  <h1>Get Involved</h1>
+<div>
```
Comment (thomasdahlberg) [inline gworv/templates/involvement/get_involved_index_page.html:9]: make this div a section tag, section tags are landmarks that help with accessibility. div tags don't have any baked in a11y benefit.

File: gworv/templates/involvement/get_involved_index_page.html [hunk 2]
```diff
@@ -1,5 +1,61 @@
...
+  <h1 id="involved-hero-header">{{ page.header }}</h1>
+  {% image page.hero_image width-400 as hero_image %}
+  <img id="involved-hero-image" src="{{ hero_image.url }}"/>
+</div>
+
+<div class="volunteer-form-container">
```
Comment (thomasdahlberg) [inline gworv/templates/involvement/get_involved_index_page.html:15]: section tag here

File: gworv/templates/involvement/get_involved_index_page.html [hunk 3]
```diff
@@ -1,5 +1,61 @@
...
+  <div class="volunteer-form-text">
+    <h4>{{ page.volunteer_header }}</h4>
```
Comment (thomasdahlberg) [inline gworv/templates/involvement/get_involved_index_page.html:17]: h2 tag here. Regardless of how big or small the font is with headers the general rule of thumb is that you only use 1 h1 tag per web page, then in descending order for other sections and subsections you go down to h2, h3, etc. In this case h2 is next in line for usage.

File: gworv/templates/involvement/get_involved_index_page.html [hunk 4]
```diff
@@ -1,5 +1,61 @@
...
+      </div>
+      <div>
+        <input type="text" name="email"/>
+      </div>
+      <input type="submit" value="Submit"/>
+    </form>
+  </div>
+
+  <div class="jobs-list-container">
+    <h4>{{ page.hiring_header }}</h4>
```
Comment (thomasdahlberg) [inline gworv/templates/involvement/get_involved_index_page.html:34]: h2 here

File: gworv/templates/involvement/get_involved_index_page.html [hunk 5]
```diff
@@ -1,5 +1,61 @@
...
+
+    <ul class="jobs-list">
+      {% for job in job_opportunities %}
```
Comment (thomasdahlberg) [inline gworv/templates/involvement/get_involved_index_page.html:37]: nice work here

File: gworv/templates/involvement/get_involved_index_page.html [hunk 6]
```diff
@@ -1,5 +1,61 @@
...
+        <li><a href="http://{{ job.link }}">{{ job.name }}</a></li>
+      {% endfor %}
+    </ul>
+  </div>
+
+  <div class="stay-involved-form-container">
```
Comment (thomasdahlberg) [inline gworv/templates/involvement/get_involved_index_page.html:43]: section tag here

File: gworv/templates/involvement/get_involved_index_page.html [hunk 7]
```diff
@@ -1,5 +1,61 @@
...
+  <div class="stay-involved-form-text">
+    <h4>{{ page.stay_involved_header }}</h4>
```
Comment (thomasdahlberg) [inline gworv/templates/involvement/get_involved_index_page.html:45]: h2 here

**Review by thomasdahlberg — COMMENTED** [review]
Just a few semantic tags to update for now. When we get to styling we might tweak it a bit more and add some aria attributes to make it slightly more accessible.

**Comment by suttondemlong** [comment]
Okie doke! @thomasdahlberg I updated the semantics for the html. Thanks for showing me how it's done!

**Review by thomasdahlberg — APPROVED** [review]

**Pull request merged**